### PR TITLE
Add Nuxt.js

### PR DESCRIPTION
**Pull request by pi0**
Link to awesome list: https://github.com/nuxt-community/awesome-nuxt

Nuxt.js is a popular versatile framework for Vue.js. (Inspired by next.js)

- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- [x] Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- [x] Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:
	- [x] **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
	- [x] It's the result of hard work and the best I could possibly produce.
	- [x] Non-generated Markdown file in a GitHub repo.
	- [x] **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- [x] The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
	- [x] Not a duplicate.
	- [x] Only has awesome items. Awesome lists are curations of the best, not everything.
	- [x] Includes a project logo/illustration whenever possible.
		- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
		- The image should link to the project website or any relevant website.
		- The image should be high-DPI. Set it to maximum half the width of the original image.
	- [x] Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
	- [x] Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
		- Should be placed on the right side of the readme heading.
		- Should link back to this list.
	- Has a Table of Contents section.
		- Should be named `Contents`, not `Table of Contents`.
		- Should be the first section in the list.
		- Should only have one level of sub-lists, preferably none.
	- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
		- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
		- If you use a license badge, it should be SVG, not PNG.
	- [x] Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
		- The file should be named `contributing.md`. Casing is up to you.
	- Has consistent formatting and proper spelling/grammar.
		- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
		- The description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
	- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- [x] Go to the top and read it again. 😋

### CODE REVIEW

**Comment by sindresorhus** [comment]
Please read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) more closely. Multiple points not followed from it. Try going through the list point for point and check that you follow it. :)

**Comment by sindresorhus** [comment]
Ping

**Comment by sindresorhus** [comment]
// @limichange

**Comment by limichange** [comment]
👌 I will follow it.

**Comment by sindresorhus** [comment]
> The image should be high-DPI. Set it to maximum half the width of the original image.

⬆️ 

---

> The description starts with an uppercase character and ends with a period.

Not followed by all of these.


---

The items in the `Official Examples` section needs to have a description.

---

The contributing guidelines are just copy-pasted from this project. They should be specific to your list. You can copy and modify this one: https://github.com/sindresorhus/awesome-nodejs/blob/master/contributing.md

File: readme.md [hunk 1]
```diff
@@ -234,6 +234,7 @@
 - [Preact](https://github.com/ooade/awesome-preact) - App framework.
 - [Progressive Enhancement](https://github.com/jbmoelker/progressive-enhancement-resources)
 - [Next.js](https://github.com/unicodeveloper/awesome-nextjs) - Framework for server-rendered React apps.
+- [Nuxt.js](https://github.com/nuxt-community/awesome-nuxt) - The Vue.js Developers Framework.
```
Comment (sindresorhus) [inline readme.md]: Needs a better description.

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**